packages/taquito-michelson-encoder/test/tokens/or.spec.ts

```diff
@@ -3,348 +3,1254 @@ import { createToken } from '../../src/tokens/createToken';
-        tokenComplexNoAnnots = createToken({ prim: 'or', args: [{ prim: 'or', args: [{ prim: 'pair', args: [{ prim: 'nat' }, { prim: 'pair', args: [{ prim: 'nat' }, { prim: 'timestamp' }] }] }, { prim: 'pair', args: [{ prim: 'pair', args: [{ prim: 'nat' }, { prim: 'mutez' }] }, { prim: 'pair', args: [{ prim: 'nat' }, { prim: 'timestamp' }] }] }] }, { prim: 'or', args: [{ prim: 'pair', args: [{ prim: 'nat' }, { prim: 'timestamp' }] }, { prim: 'or', args: [{ prim: 'pair', args: [{ prim: 'nat' }, { prim: 'pair', args: [{ prim: 'mutez' }, { prim: 'timestamp' }] }] }, { prim: 'nat' }] }] }] }, 0) as OrToken;
-        tokenNestedOr = createToken({ "prim": "or", "args": [{ "prim": "or", "args": [{ "prim": "or", "args": [{ "prim": "address", "annots": ["%myAddress"] }, { "prim": "bytes", "annots": ["%myBytes"] }] }, { "prim": "or", "args": [{ "prim": "int", "annots": ["%myInt"] }, { "prim": "nat", "annots": ["%myNat"] }] }] }, { "prim": "or", "args": [{ "prim": "or", "args": [{ "prim": "pair", "args": [{ "prim": "nat" }, { "prim": "int" }], "annots": ["%myPair"] }, { "prim": "string", "annots": ["%myString"] }] }, { "prim": "mutez", "annots": ["%myTez"] }] }] }, 0) as OrToken;
-        tokenNestedOrWithoutAnnot = createToken({ "prim": "or", "args": [{ "prim": "or", "args": [{ "prim": "or", "args": [{ "prim": "address" }, { "prim": "bytes" }] }, { "prim": "or", "args": [{ "prim": "int" }, { "prim": "nat" }] }] }, { "prim": "or", "args": [{ "prim": "or", "args": [{ "prim": "pair", "args": [{ "prim": "nat" }, { "prim": "int" }] }, { "prim": "string" }] }, { "prim": "mutez" }] }] }, 0) as OrToken;
-        tokenOrWithOption = createToken({ prim: 'or', args: [{ prim: 'int' }, { prim: 'or', args: [{ prim: 'nat' }, { prim: 'or', args: [{ prim: 'mutez' }, { prim: 'option', args: [{ prim: 'or', args: [{ prim: 'int' }, { prim: 'string' }], annots: [] }], annots: [] }], annots: [] }], annots: [] }], annots: [] }, 0) as OrToken;
-    });
+  let token: OrToken;
+  let tokenNoAnnots: OrToken;
+  let tokenComplex: OrToken;
+  let tokenComplexNoAnnots: OrToken;
+  let tokenNestedOr: OrToken;
+  let token2LevelOrMixedAnnots: OrToken;
+  let token3LevelOrMixedAnnots: OrToken;
+  let tokenNestedOrWithoutAnnot: OrToken;
+  let tokenOrWithOption: OrToken;
+  beforeEach(() => {
+    token = createToken(
+      {
```

Comment:
suggestion only, i found the format of json Michelson in one line will be easier to comprehend what this test is really about.